app/src/main/java/com/github/sdpteam15/polyevents/model/database/local/room/LogAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.sdpteam15.polyevents.model.database.local.room
+
+import com.github.sdpteam15.polyevents.helper.HelperFunctions
+import com.github.sdpteam15.polyevents.model.database.remote.adapter.AdapterFromDocumentInterface
+import com.github.sdpteam15.polyevents.model.database.remote.adapter.AdapterToDocumentInterface
+import java.time.LocalDateTime
+import java.util.*
+
+object LogAdapter {
+    const val LAST_UPDATE = "LAST_UPDATE"
+    const val IS_VALID = "IS_VALID"
+}
+
+/**
+ * Add Log to an entity on the database from a other adapter
+ * @param adapter the adapter
+ */
+class LogAdapterToDocument<T>(private val adapter: AdapterToDocumentInterface<T>) :
```

Comment:
Maybe add more explanation what this does?

Reply:
Yes, will be don in my next PR